### ISSUE TITLE
Extensions are not being booted in lumen

### ISSUE THREAD

**Issue by Edwin-Luijten** [laravel-doctrine/orm#84]
``` php
$this->app['events']->listen('router.matched', function () {
            $this->app->make(ExtensionManager::class)->boot();
        });
```

Lumen 5.1 does not contain the event 'router.matched' it's only available in Illuminate\Router
therefore it extensions are not being booted in Lumen.

I also find it a rather strange "location" to boot the extensions when router.matched is fired, is there a reason for it?

By placing the booting of the extension manager outside the listener it's booting the extensions again.

**Comment by FoxxMD** [comment]
`fastRoute` (lumen's router) does not broadcast any events either so hooking into the router doesn't seem viable. 

As suggest by @Edwin-Luijten we could require users boot the extension manager themself or

Could we potentially move the boot process for the extension manager into the gedmo SP?

**Comment by patrickbrouwers** [comment]
The extension manager has to be booted after the session is started. It's the only event I could find.
If it's booted before the session is started, Loggable won't work (Auth::user() is null)

**Comment by ascirpus** [comment]
:+1: to this issue.

While I see the reasoning that @patrickbrouwers mentions - currently it's either kill Lumen entirely or kill Loggable.

You can't attach it to events because it won't work in Lumen.
We could throw out loggable for Lumen, because there's no Guard in Lumen anyway, revert back to the way it's been working.
And specifically for Laravel you could load Loggable specifically on auth.login event. 
Would that work?

**Comment by patrickbrouwers** [comment]
Wish Laravel was a bit more consistent. But in the mean-time we have found a work-around. Will be pushed today.

**Comment by maxbrokman** [comment]
Not had time to test my work on this today unfortunately, but soon!

**Comment by patrickbrouwers** [comment]
@nexsja btw Lumen does have Guard. And it does support Sessions (if you enable the middleware)

**Comment by patrickbrouwers** [comment]
Fixed in 1.0.9
